src/test/scala/nl/knaw/dans/easy/bagindex/server/BagIndexServletSpec.scala

```diff
@@ -282,16 +282,16 @@ class BagIndexServletSpec extends TestSupportFixture
     }
   }
 
-  it should "fail when the bag is not found in the bagstore" in {
+  it should "fail when the bag is not found in the bagstore, returning not found" in {
     val uuid = UUID.randomUUID()
 
     put(s"/bags/$uuid") {
-      status shouldBe 400
+      status shouldBe 404
       body shouldBe s"The bag with id '$uuid' could not be found"
     }
   }
 
-  it should "fail when the parameter 'bagId' is not present" in {
+  it should "fail when the parameter 'bagId' is not present" in { //TODO isn't this a 400??
```

Comment:
Good catch!!! You are completely right! This should be a 400. Please fix it in the same PR and add this change to the cover message as well.

Reply:
I have looked a gain and there is no handler for the end point ```/bags```. So perhaps it should stay a 404 since there is no handler for this curl?

Reply:
@rvanheest , @stephanHogenboom : no, this is *not* a 400. 404 is better. You *could* argue that the URI does exist, as it is the parent of other URI's. In that case the status code would be "405 Method not allowed". However the `/bags` endpoint doesn't support any other method, either, so we might as well say it doesn't exist, which seems to be Scalatra's default behavior. In any case it is not a 400. 400 would mean the message is malformed, which is not the problem here.  I think the current behavior is good enough.